### ISSUE TITLE
Attention mask always returns array of ones for CamembertTokenizer.batch_encode_plus

### ISSUE THREAD

**Issue by Wissben** [huggingface/transformers#3203]
# 🐛 Bug

## Information

The model i'm using is CamebertTokenizer for the French language

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [x] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below)

## To reproduce

- First init the models with the following code:
```python
camembert_tokenizer = CamembertTokenizer.from_pretrained('../models/', cache_dir='./models')
camembert_tf_model = TFCamembertModel.from_pretrained('../models/', output_hidden_states=True,
                                                          cache_dir='./models'
                                                     )
camembert_tf_model.trainable = False
```
- Prepare the input data :
```python
text = ' '.join(['je' for i in range(25)])
texts = [ text, "je suis cool"]
input_ids = camembert_tokenizer.batch_encode_plus(texts,
                                             add_special_tokens=True,
                                             max_length=8,
                                            return_tensors='tf')
print(input_ids)
```
## Expected behavior

What should happen is that the padded tokens should have a mask value of 0 if I've correctly understood the doc. so the output of the snippet should be : 
```
{'input_ids': <tf.Tensor: shape=(2, 8), dtype=int32, numpy=
 array([[   5,   50,   50,   50,   50,   50,   50,    6],
        [   5,   50,  146, 4261,    6,    1,    1,    1]], dtype=int32)>,
 'token_type_ids': <tf.Tensor: shape=(2, 8), dtype=int32, numpy=
 array([[0, 0, 0, 0, 0, 0, 0, 0],
        [0, 0, 0, 0, 0, 1, 1, 1]], dtype=int32)>,
 'attention_mask': <tf.Tensor: shape=(2, 8), dtype=int32, numpy=
 array([[1, 1, 1, 1, 1, 1, 1, 1],
        [1, 1, 1, 1, 1, 0, 0, 0]], dtype=int32)>}
```
Instead, i'm always getting an attention_mask full of ones like this : 
```
{'input_ids': <tf.Tensor: shape=(2, 8), dtype=int32, numpy=
 array([[   5,   50,   50,   50,   50,   50,   50,    6],
        [   5,   50,  146, 4261,    6,    1,    1,    1]], dtype=int32)>,
 'token_type_ids': <tf.Tensor: shape=(2, 8), dtype=int32, numpy=
 array([[0, 0, 0, 0, 0, 0, 0, 0],
        [0, 0, 0, 0, 0, 1, 1, 1]], dtype=int32)>,
 'attention_mask': <tf.Tensor: shape=(2, 8), dtype=int32, numpy=
 array([[1, 1, 1, 1, 1, 1, 1, 1],
        [1, 1, 1, 1, 1, 1, 1, 1]], dtype=int32)>}
```

## Environment info
- `transformers` version: 2.5.1
- Platform: Linux-5.3.0-40-generic-x86_64-with-Ubuntu-18.04-bionic
- Python version: 3.6.9
- PyTorch version (GPU?): 1.4.0 (False)
- Tensorflow version (GPU?): 2.1.0 (False)
- Using GPU in script?: (False)
- Using distributed or parallel set-up in script?: (False)

**Comment by Wissben** [comment]
**EDIT** : 
I've been accidentally testing this code on the 2.4.1 version in a different environment. Since I've updated to 2.5.1 the behavior is as expected.
